products/sle/main.pm

```diff
@@ -380,7 +380,10 @@ sub load_x11regression_documentation {
         loadtest "x11regressions/libreoffice/libreoffice_mainmenu_components";
         loadtest "x11regressions/libreoffice/libreoffice_recent_documents";
         loadtest "x11regressions/libreoffice/libreoffice_default_theme";
-        loadtest "x11regressions/libreoffice/libreoffice_pyuno_bridge";
+        loadtest "x11regressions/evolution/evolution_prepare_servers";
+        if (get_var("VERSION") =~ /12/) {
```

Comment:
Sorry, I don't understand. At first you can also turn around by saying `! sle_version_at_least` or implement `sle_version_at_most` easily. Also, if we assume that in general the same module should also be supported on sle15, simply make it `sle_version_at_least('15')` and we will need to take care as soon as we have a sle15 test that fails accordingly